dc/core/DisplayGroup.cpp

```diff
@@ -203,29 +204,36 @@ void DisplayGroup::unfocus( const QUuid& id )
         return;
 
     window->setFocused( false );
-    removeFocusedWindow( window );
+    _removeFocusedWindow( window );
     // Make sure the window dimensions are re-adjusted to the new zoom level
     window->getController()->scale( window->getCoordinates().center(), 0.0 );
+    _updateFocusedWindowsCoordinates();
+
     sendDisplayGroup();
 }
 
+const std::set<ContentWindowPtr>& DisplayGroup::getFocusedWindows() const
```

Comment:
OK, I will see how I do that. A set is a more correct container than a vector since we don't want duplicates. Maybe I can change the typedef of ContentWindowPtrs as well because I don't think we want to have duplicates anywhere.